src/api/msServices/msServices.service.v6.js

```diff
@@ -22,6 +22,14 @@ angular
       edit: {
         method: 'PUT', cache, isArray: false, interceptor,
       },
+      createMfaOnAllUsers: {
+        method: 'POST',
+        url: '/msServices/:serviceName/createMfaOnAllUsers ',
```

Comment:
`interception` isn't needed in your case?

Reply:
No, because `createMfaOnAllUsers` won't affect the `service`, it create `mfa` on sub-`account`s